projects/scion/microfrontend-platform/src/lib/host/message-broker/message-interception.ts

```diff
@@ -108,11 +109,14 @@ export abstract class IntentInterceptor implements Interceptor<IntentMessage, Ha
    * or to swallow the intent by not calling the next handler at all. If rejecting publishing, the error is transported to
    * the intent issuer.
    *
+   * Important: if you continue publishing by passing the message to the next handler, make sure to either return the resulting Promise,
```

Comment:
Consider rewriting the comment as follows:
**Important**: When passing the message to the next handler, either return its Promise or `await` it. Otherwise, errors of subsequent interceptors would not be reported to the sender.

Reply:
done